exercises/variables/variables3.d

```diff
@@ -0,0 +1,25 @@
+/// Variables in D are mutable by default. But sometimes we want to express
+/// them otherwise and make their values unchangeable.
+
+/// There are 3 ways to make this happen:
+/// * enum constants
+/// * const
+/// * immutable
+
+/// Working with constant values also allows us to work with CTFE - Compile Time
+/// Function Evaluation - a D feature that allows any function to execute at
+/// compile time! We will get more into this in future chapters.
+
+/// Since constant values cannot change, once their initialized their state
```

Comment:
```suggestion
/// Since constant values cannot change, once they're initialized, their state
```

Reply:
Fixed in 87026fe893fc714498c9f85d1ea33b9a3996f3af